spec/requests/stories_index_spec.rb

```diff
@@ -159,12 +159,27 @@
         expect(response.body).not_to include(CGI.escapeHTML("Super-sheep"))
       end
 
-      it "displays posts with the campaign tags when sidebar is enabled" do
+      it "doesn't display low-score posts" do
         SiteConfig.campaign_sidebar_enabled = true
+        SiteConfig.campaign_articles_require_approval = true
         get "/"
         expect(response.body).not_to include(CGI.escapeHTML("Unapproved-post"))
       end
 
+      it "doesn't display unapproved posts" do
+        SiteConfig.campaign_sidebar_enabled = true
+        SiteConfig.campaign_articles_require_approval = true
+        Article.last.update_column(:score, -2)
+        get "/"
+        expect(response.body).not_to include(CGI.escapeHTML("Unapproved-post"))
+      end
+
+      it "displays unapproved post if approval is not required" do
+        SiteConfig.campaign_sidebar_enabled = true
+        get "/"
+        expect(response.body).to include(CGI.escapeHTML("Unapproved-post"))
```

Comment:
What I mean by that other inline comment is that the spec will fail if we add the following line here:

`expect(response.body).to include(CGI.escapeHTML("Super-sheep"))`

Which I think makes sense: "If we don't need approval with the `SiteConfig` I should be able to see Posts whether they're approved or not"